File: Computer-Seekho-.NET/Computer_Seekho_DN/Models/GetInTouch.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace Computer_Seekho_DN.Models;

[Table("get_in_touch")]
public partial class GetInTouch
{
    [Key]
    [Column("get_in_touch_id")]
    public int GetInTouchId { get; set; }

    [Column("course_name")]
    [StringLength(255)]
    public string? CourseName { get; set; }

    [Column("enquirer_email")]
    [StringLength(255)]
    public string EnquirerEmail { get; set; } = null!;

    [Column("enquirer_name")]
    [StringLength(255)]
    public string? EnquirerName { get; set; }

    [Column("enquirer_message")]
    [StringLength(255)]
    public string? EnquirerMessage { get; set; }

    [Column("enquirer_mobile")]
    [StringLength(255)]
    public string EnquirerMobile { get; set; } = null!;
}

File: Computer-Seekho-Java/ComputerSeekho/src/main/java/com/Project/Entities/GetInTouch.java
package com.Project.Entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Pattern;
import lombok.Data;


@Entity
@Data
@Table(name = "get_in_touch")
public class GetInTouch {
    
    @Id
    @Column(name = "get_in_touch_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int getInTouchId;

    @Column(name = "enquirer_name")
    private String enquirerName;

    @Column(name = "enquirer_email")
    @NotBlank(message = "Email is required")
    @Email(message = "Email must be valid")
    private String email;

    @NotBlank(message = "Mobile number is required")
    @Pattern(regexp = "\\d{10}", message = "Mobile number must be 10 digits")
    @Column(name = "enquirer_mobile")
    private String mobile;

    @Column(name = "enquirer_message")
    private String enquiryMessage;
    
    @Column(name = "course_name")
    private String courseName;
}
